### ISSUE TITLE
DADGAD?

### ISSUE THREAD

**Issue by rtxanson** [pcorey/chord#1]
More of an idea, but I wonder about doing this for alternate tunings too. I've watched some DADGAD players who seem to already be doing minimal effort chord changes. Might be interesting to know if you can quantify whether a tuning has less effort involved for certain keys, or something.

**Comment by pcorey** [comment]
Yeah, cool idea! The pitches for standard tuning are actually partially hard-coded [a default function argument here](https://github.com/pcorey/chord/blob/master/lib/chord/voicing.ex#L75). Unfortunately, that's a private function and that argument it's exposed further up the call stack. I should add the ability to override the tuning. That would be a nice feature.

**Comment by jem-computer** [comment]
@pcorey any progress on supporting alternate tunings? would be super rad to specify them in ['Joni tuning notation'](http://jonimitchell.com/music/notation.cfm)

(also thank you for this repo - I'm in love)

**Comment by pcorey** [comment]
@jongold Thanks Jon! I haven't given alternate tunings any real attention yet. This repo's actually undergoing some serious changes behind the scenes, but I'll keep that in mind for the next version.

I've never heard of Joni notation. That's a pretty cool idea. Thanks!

**Comment by rtxanson** [comment]
Yeah, same. Joni Mitchell is even more legendary now!
